src/selectors/teams.js

```diff
@@ -0,0 +1,27 @@
+import { createSelector } from 'reselect';
+
+/**
+ * @description Return team list from state
+ * @param {Object} state app state
+ */
+const getTeams = state => state.teams.list;
+
+/**
+ * @description selector for team array
+ * @return {Array} array of teams objects
+ */
+export const getTeamsArray = createSelector(getTeams, (teams) => {
+  const arr = [];
+  teams.forEach((team, id) => {
```

Comment:
- Firebase API client first checks data in localStorage and then makes fetch and update data. During that process, it doesn't return Map() instance but JS Object. Anyway, It should not be happening anymore. Let me know if the problem persists
- I've moved DirectSort to header
- You should update dependencies again. Please, don't forget it